minetest/minetest
# [no squash] Auto-toggle TouchScreenGUI in-game when receiving touch/mouse input

With this PR, the touch controls are automatically created/deleted based on the last used pointer type (mouse or touch).

Note that no automatic switching is done on keyboard input. This is because TouchScreenGUI + keyboard works while TouchScreenGUI + mouse doesn't.

Related: #14400, https://github.com/minetest/minetest/pull/14400#issuecomment-1959930556, #14749

## To do

This PR is a Ready for Review.

## How to test

Buy an Android phone, a bluetooth mouse and a bluetooth keyboard. See that the touch controls are automatically created/deleted when you use the touchscreen/mouse.
